pkg/clientgen/testdata/tsapp/expected_list_of_union_shared.ts

```diff
@@ -105,10 +108,14 @@ export namespace svc {
 
 type PickMethods<Type> = Omit<CallParameters, "method"> & {method?: Type}
 
+type OmitCookie<T> = {
```

Comment:
If all keys are cookies, this will yield the type `{}` which suggests it's still a value and not `void`. Is that how the backend behaves, or should the type evaluate to `void` in that case? (Annoying questions, I know)

Reply:
good question :thinking: when we have shared types turned off the types get rendered as `{}` as well, and I think that works actually, but maybe it would be cleaner to do it the same way as if there where no type, I'll see what I can come up with for both shared and not shared types :eyes: